base.cmake

```diff
@@ -210,14 +210,8 @@ MACRO(SETUP_PROJECT)
     # Be verbose by default.
     SET(CMAKE_VERBOSE_MAKEFILE TRUE)
   ENDIF(${ARGC})
-  
-  # If the variable INSTALL_DOCUMENTATION is not set, then the project will install the doc. Otherwise, 
-  # the doc will be installed if INSTALL_DOCUMENTATION is set to TRUE.
-  IF(DEFINED INSTALL_DOCUMENTATION)
-    SET(_INSTALL_DOC ${INSTALL_DOCUMENTATION} CACHE INTERNAL "")
-  ELSE(DEFINED INSTALL_DOCUMENTATION)
-    SET(_INSTALL_DOC TRUE CACHE INTERNAL "")
-  ENDIF(DEFINED INSTALL_DOCUMENTATION)
+
+  OPTION(INSTALL_DOCUMENTATION "Generate and install the documentation" ON)
```

Comment:
Open question: should we keep *INSTALL_DOCUMENTATION* or should we use *PROJECT_*... ?

Reply:
As we are already using this in many places, it would be easier for our projects not to change it. Apart from that, I have no preference towards one ore the other

Reply:
Certainly `PROJECT_INSTALL_DOCUMENTATION`will never conflict with Cmake.

Reply:
On the other hand, if CMake ever provides an option for this, we'd better use it

Reply:
> On the other hand, if CMake ever provides an option for this, we'd better use it

Yes, but that takes some time and the hypothetical CMake method might not be compatible with our method.

Reply:
So, to conclude this conversation, the better is to keep `INSTALL_DOCUMENTATION`. We will change later if really needed.